src/core/captors/touch.ts

```diff
@@ -95,7 +96,6 @@ export default class TouchCaptor extends Captor<TouchCaptorEvents> {
     if (e.touches.length === 1) this.dispatchRelatedMouseEvent("mousedown", e);
 
     const touches = getTouchesArray(e.touches);
-    this.isMoving = true;
```

Comment:
this line removal is unrelated to the PR in the end but it makes sense that isMoving would only be true after an actual move than immediately after touchdown